src/libs/actions/IOU.ts

```diff
@@ -326,7 +327,8 @@ function initMoneyRequest(reportID: string, policy: OnyxEntry<OnyxTypes.Policy>,
     });
 }
 
-function clearMoneyRequest(transactionID: string) {
+function clearMoneyRequest(transactionID: string, skipConfirmation = false) {
```

Comment:
missing the type for skipconfirmation

```suggestion
function clearMoneyRequest(transactionID: string, skipConfirmation: boolean = false) {
```

Reply:
I think we don't need the type when there is a default

Reply:
I could not find that convention here https://github.com/Expensify/App/blob/main/contributingGuides/TS_STYLE.md

Reply:
Fair enough. Adding them 😁

Reply:
Nope. Removing them:
<img width="969" alt="Screenshot 2024-04-16 at 4 30 42 PM" src="https://github.com/Expensify/App/assets/19366280/f5ea7aef-dbaa-4dc6-9f84-e7437d52c4f0">